### ISSUE TITLE
no working on oculus quest 2

### ISSUE THREAD

**Issue by sholms** [FullStackIndie/EasyCodeForVivox#2]
hello, I was trying this asset on oculus quest 2, I tried the sample scene but the voice chat don't work, but when I try the Vivox sample scene it work fine, do you know what is causing this problem?

**Comment by FullStackIndie** [comment]
Yes for some reason the demo scene does not work. I have updated the asset on the Unity Asset Store but not on Github yet. I really should tho lol. You can find the latest version here
https://assetstore.unity.com/packages/add-ons/easycodeforvivox-202190

I will try and update this code and keep it up to date with asset store realesses and pre-releases

**Comment by sholms** [comment]
Hi, sorry for bothering you but the version from the unity asset store have the same problem :c

**Comment by FullStackIndie** [comment]
Its all good man. Is it the latest version 1.3. Sometimes Unity doesn't auto update the package if you have downloaded it before. If that's not the problem, then you have to make your own custom scene and just use the EasyExample script to access Vivox functionality. Is oculus for mobile? you might also have to request mic access if working with android or ios
